app/main/posts/detail/post-detail-data.directive.js

```diff
@@ -83,6 +87,12 @@ function PostDetailDataController(
             });
         }
 
+        // Route and display accordingly if the we are in mobile or tablet view
+        let currentWidth = $window.innerWidth;
+        if (currentWidth < 1023) {
```

Comment:
One issue with this is that if the user resizes the window after the initial load this won't have been run for them, yielding weird results. We should probably run this for everyone and always set the active classes.